app/Mail/ContactUs.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Attachment;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class ContactUs extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * @var array<string, string>
+     */
+    public array $data;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param array<string, string> $data
+     */
+    public function __construct(array $data)
+    {
+        $this->data = $data;
+    }
+
+    /**
+     * Get the message envelope.
+     */
+    public function envelope(): Envelope
+    {
+        return new Envelope(
+            subject: 'Contact Us | ' . $this->data['subject'],
```

Comment:
This way you can make it always have the first letter of the subject as uppercase

```suggestion
            subject: 'Contact Us | ' . ucfirst($this->data['subject']),
```